secrets/secrets.go

```diff
@@ -31,19 +31,28 @@ const (
 
 	// NetworkKey is the libp2p private key secret used for networking
 	NetworkKey = "network-key"
+
+	// JSONTLSKey is the tls private key used for json rpc https endpoint
+	JSONTLSKey = "jsontls-key"
+
+	// JSONTLSCert is the tls certificate used for json rpc https endpoint
+	JSONTLSCert = "jsontls-pem"
 )
 
 // Define constant file names for the local StorageManager
 const (
 	ValidatorKeyLocal    = "validator.key"
 	ValidatorBLSKeyLocal = "validator-bls.key"
 	NetworkKeyLocal      = "libp2p.key"
+	JSONTLSKeyLocal      = "/private/jsontls.key"
+	JSONTLSCertLocal     = "/certs/jsontls.pem"
 )
 
 // Define constant folder names for the local StorageManager
 const (
 	ConsensusFolderLocal = "consensus"
 	NetworkFolderLocal   = "libp2p"
+	JSONTLSFolderLocal   = "/etc/ssl"
```

Comment:
Certificate must go to this folder location, right? It would not work correctly if we put it alongside with other secrets (such as ECDSA key, lip2p key etc.)?